packages/components/modal/src/docs/osds-modal/usage.mdx

```diff
@@ -5,7 +5,22 @@ import GenericStyle from '@ovhcloud/ods-common-core/docs/generic-style.mdx';
 ## Usage
 
 ### Default
-<osds-modal>Modal</osds-modal>
+
+export const buttonTrigger = () => {
+  setTimeout(() => {
```

Comment:
why the setTimeout ?

Reply:
It's because the render of the page is not finish otherwise the selector is undefined

Reply:
can't we do it with an event listener of `DOMContentLoaded` instead ?

Reply:
I tried but it didn't lead to success.